thetis/shallowwater_eq.py

```diff
@@ -317,8 +329,7 @@ def residual(self, uv, eta, uv_old, eta_old, fields, fields_old, bnd_conditions=
                     # assume land boundary
                     # impermeability implies external un=0
                     un_jump = inner(uv, self.normal)
-                    h = self.bathymetry
-                    head_rie = head + sqrt(h/g_grav)*un_jump
+                    head_rie = head + sqrt(total_h/g_grav)*un_jump
```

Comment:
Assuming that our test coverage is good enough, passing all the tests should be sufficient. So the first thing to do is to fix the lint issues, so that drone can run the test suite.